src/presence_analyzer/views.py

```diff
@@ -75,3 +81,24 @@ def presence_weekday_view(user_id):
 
     result.insert(0, ('Weekday', 'Presence (s)'))
     return result
+
+
+@app.route('/api/v1/presence_start_end/<int:user_id>', methods=['GET'])
+@jsonify
+def presence_start_end_view(user_id):
+    """
+    Returns the mean time interval of the user's presence grouped by weekday.
+    """
+    data = get_data()
+    if user_id not in data:
+        log.debug('User %s not found!', user_id)
+        abort(404)
+
+    weekdays = group_start_end_by_weekday(data[user_id])
+    days = [
+        (calendar.day_abbr[weekday], (times[0]), times[1])
+        for weekday, times in enumerate(weekdays)
```

Comment:
Not like i run it yet, but seems we're misusing enumerate to generate weekdays. Correct me if im wrong but `weekdays` var already has both of it, aye?

Reply:
`weekdays` var is like `[[int], [int]]`, we need to use enumerate to generate number for `calendar.day_abbr`. I wrote this based on the previous views